mfr/extensions/tabular/render.py

```diff
@@ -41,7 +42,14 @@ def _render_grid(self, fp, ext, *args, **kwargs):  # assets_path, ext):
         """
         self._renderer_tabular_metrics = {}
 
-        sheets = self._populate_data(fp, ext)
+        try:
+            sheets = self._populate_data(fp, ext)
+        except _csv.Error as e:
+            if any("field larger than field limit" in errorMsg for errorMsg in e.args):
+                raise exceptions.TabularRendererError('This file contains a field too large to render. '
```

Comment:
`TabularRendererError` throws a 500 by default (well, one of its parents does), but I think a 40x would be more appropriate for this particular error.